assembl/static2/js/app/pages/synthesis.jsx

```diff
@@ -48,7 +108,10 @@ export class DumbSynthesis extends React.Component<void, SynthesisProps, void> {
               </Section>
             )}
             <Row className="background-grey synthesis-tree">
-              <Col mdOffset={3} md={7} smOffset={1} sm={10}>
+              <Col md={3} className="affix" style={{ top: sideMenuTop }}>
+                {!isHidden && <SideMenu rootIdeas={roots} descendants={descendants} synthesisPostId={synthesisPostId} />}
```

Comment:
`isHidden` is too generic (maybe use something like `displaySideMenu` or `sideMenuIsHidden`)